archive/src/main/java/io/orkes/conductor/dao/indexer/IndexValueExtractor.java

```diff
@@ -59,9 +59,15 @@ private static List<String> getIndexWords(WorkflowModel workflow) {
         append(words, workflow.getReasonForIncompletion());
         append(words, workflow.getVariables());
 
+        if (workflow.getParentWorkflowId() == null) {
+            String message = "parent_wf";
```

Comment:
I would probably name this "root_wf" instead

and also extract it to a constant